packages/webapp/src/components/FilterDropdown/Markup.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { MenuItem, CssBaseline } from '@material-ui/core';
+
+import { FormContainer, SelectPreview, CircularProgressStyled } from './styled';
+
+const callSpinner = () => <CircularProgressStyled size={20} thickness={2.5} />;
+
+const callMenuItems = (options, loading) => {
+  if (loading) {
+    return (
+      <MenuItem value="boom" displayEmpty name="something">
+        {callSpinner()}
+      </MenuItem>
+    );
+  }
+
+  return options.map(({ id: idVal, name }) => {
+    const selectedKey = options.findIndex(({ id }) => id === idVal);
+
+    return (
+      <MenuItem key={selectedKey} value={idVal}>
+        {name}
+      </MenuItem>
+    );
+  });
+};
+
+const callOptions = (options, selected, eventHandler, greyTheme, disabled, loading) => (
+  <SelectPreview
+    {...{ greyTheme }}
```

Comment:
What does `greyTheme` do? Perhaps just add a prop to a component called `primary` which is a boolean. If `primary` is true then apply the white styling.

Reply:
Swapped to `primary`